pallet-logion-loc/src/mock.rs

```diff
@@ -92,6 +92,31 @@ parameter_types! {
     pub const MaxTokensRecordFiles: u32 = 10;
 }
 
+// Type used as beneficiary payout handle
+pub struct RewardDistributorImpl();
+impl RewardDistributor<NegativeImbalanceOf<Test>, Balance>
+for RewardDistributorImpl
+{
+    fn payout_reserve(_reward: NegativeImbalanceOf<Test>) {
+    }
+
+    fn payout_collators(_reward: NegativeImbalanceOf<Test>) {
+    }
+
+    fn payout_stakers(_reward: NegativeImbalanceOf<Test>) {
+    }
```

Comment:
What about actually doing something with the reward and test that it was properly distributed? This could be done in the unit test addressing the successful addition of a file to a LOC.

Reply:
Agreed, this should be tested.
A [task](https://github.com/logion-network/logion-internal/issues/819) is created for that.